tests/misc/errors/babel-ts/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,8 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`issue-8041.ts error test 1`] = `
-"Did not expect a type annotation here. (1:2)
-> 1 | (a:b)
-    |  ^
+exports[`type-annotation.ts error test 1`] = `
+"Did not expect a type annotation here. (1:9)
+> 1 | func<T>(a: T);
+    |         ^
```

Comment:
Why does Babel parse `(a:b)` without errors now?

Reply:
Okay, I found https://github.com/babel/babel/pull/12185
So this test should stay.